docs/deployment/digital-ocean-apps.mdx

```diff
@@ -237,6 +237,324 @@ Done! Application deployed and can be accessed by provided domain.
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: registry.digitalocean.com
+          username: ${{ secrets.DO_USERNAME }}
+          password: ${{ secrets.DO_ACCESS_TOKEN }}
+
+      - name: Build and Push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64
+          file: ${{ matrix.dockerfile }}
+          tags: registry.digitalocean.com/ship-demo/ship-demo-${{ matrix.app_name }}:${{ github.sha }}
+          target: runner
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+```
+
+**WEB Build and Push Docker Image GH Action:**
+```yaml
+name: Build and Push Staging Web Image to DigitalOcean
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "apps/web/**"
+      - "packages/**"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build:
+    name: Build and Push Docker Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: registry.digitalocean.com
+          username: ${{ secrets.DO_USERNAME }}
+          password: ${{ secrets.DO_ACCESS_TOKEN }}
+
+      - name: Build and Push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64
+          file: ./apps/web/Dockerfile
+          tags: registry.digitalocean.com/ship-demo/ship-demo-web:${{ github.sha }}
+          target: runner
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+```
+
+3. Add 3 new github secrets: `DO_USERNAME` (the email of your DigitalOcean account), `DO_WEB_STAGING_APP_NAME` (the name of the web app) and `DO_API_STAGING_APP_NAME` (the name of the API app). And run 2 new workflows manually to push the initial images to the DigitalOcean Container Registry.
+
+![Create new github secrets for DO container registry](/images/do-registry-github-secrets.png)
+
+![Run and Build Docker images result](/images/run-build-and-push-do-image-results.png)
+
+4. Navigate to your DigitalOcean Container Registry. You should see 4 newly added repositories, as shown in the screenshot below.
+
+![Pushed Docker Images in DOCR](/images/pushed-images-in-docr.png)
+
+5. Navigate to Application Spec `(settings tab)`. Remove `dockerfile_path`, `github`, `source_dir` variables for migrator, scheduler and api. Migrator is placed in the `jobs` section. The scheduler is placed in the `workers` section. The api is placed in the `services` section. You can also find it by name of the resource.
```

Comment:
try to use mintlify code component with diff (https://www.mintlify.com/docs/create/code#diff) for showing which lines in app settings where updated 

also make all **large** configs [expandable](https://www.mintlify.com/docs/create/code#expandable)